### PR TITLE
Add Homebrew and Codesign notes for macOS

### PR DESCRIPTION
**Pull request by andyleejordan**
This documents the installation of Homebrew and setup for group use, and an extra link on how to codesign GDB (although this is already provided upon installation of GDB).

### CODE REVIEW

File: setup-macosx.md [hunk 1]
```diff
@@ -182,6 +184,36 @@ cd updatedns
...
+```sh
+sudo chgrp -R staff /usr/local
+sudo chmod -R g+w /usr/local
+```
+
+[homebrew]: http://brew.sh/
+
+#### Codesign GDB
+
+Per the Homebrew installation notes, "gdb requires special privileges to access Mach ports.
```
Comment (jeffaco) [inline setup-macosx.md:210]: Missing " character (unmatched ").

File: setup-macosx.md [hunk 2]
```diff
@@ -182,6 +184,36 @@ cd updatedns
...
+You will need to codesign the binary. For instructions, see:" [BuildingOnDarwin](https://sourceware.org/gdb/wiki/BuildingOnDarwin).
```
Comment (jeffaco) [inline setup-macosx.md:211]: Ditto with above, unmatched "

File: setup-macosx.md [hunk 3]
```diff
@@ -182,6 +184,36 @@ cd updatedns
...
+
+Since our macOS systems are shared, please check if a GDB certificate has already been created.
+
+Codesigning GDB needs to be done each time GDB updates, XCode updates, or the certificate expires.
+
 #### Creating a New Virtual Machine
```
Comment (jeffaco) [inline setup-macosx.md:217]: While you're in here, can you remove one # character (put it at same level as "Setup"). That is, this should have three # characters, not four. Thanks.

**Review by jeffaco — APPROVED** [review]
Please apply my feedback, them merge. Thanks.

File: setup-macosx.md (hunk 2 above)
Comment (andyleejordan) [reply setup-macosx.md:211]: The two lines are part of the same quote.

File: setup-macosx.md (hunk 3 above)
Comment (andyleejordan) [reply setup-macosx.md:217]: I didn't add this line.

File: setup-macosx.md (hunk 1 above)
Comment (andyleejordan) [reply setup-macosx.md:210]: It's matched on the next line.

**Pull request merged**